Documentation/contributing/testing/unit.rst

```diff
@@ -63,6 +63,31 @@ run the unit tests and tear down the prerequisites:
 
     $ make unit-tests TESTPKGS=github.com/cilium/cilium/pkg/kvstore
 
+There are some 'privileged' tests in some packages. Those privileged tests are
+not run by default when you run the unit tests for the respective package.
+The privileged test files have an entry at the top of the test file as shown.
+
+::
+
+    +build privileged_tests
+
+There are two ways that you can run the 'privileged' tests. All privileged tests
+have to be run as 'root' user.
+
+1. To run all the 'privileged' tests for cilium follow the instructions below.
+
+::
+
+    $ sudo make tests-privileged
```

Comment:
```suggestion
    $ sudo -E make tests-privileged
```
Encouraging the use of `-E` I think is an small, but important thing to do. :)

Reply:
Done